rq/worker.py

```diff
@@ -85,6 +85,13 @@ def signal_name(signum):
 )
 
 
+def move_to_failed_queue(job, *exc_info):
```

Comment:
Thanks for the PR. To keep things tidy, could you please move this function to `handlers.py`?